release/kubernetes.yaml

```diff
@@ -41,22 +44,20 @@ spec:
           allowPrivilegeEscalation: false
           capabilities:
             drop:
-              - all
+              - ALL
           privileged: false
           readOnlyRootFilesystem: true
-        image: gcr.io/google-samples/microservices-demo/emailservice:v0.3.9
+        image: gcr.io/google-samples/microservices-demo/emailservice:v0.6.0
         ports:
         - containerPort: 8080
         env:
         - name: PORT
           value: "8080"
-        - name: DISABLE_TRACING
-          value: "1"
         - name: DISABLE_PROFILER
           value: "1"
         resources:
           requests:
-            cpu: 10m
+            cpu: 100m
```

Comment:
Let's reduce the resource so we can have more deployments in a smaller cluster. Actually I prefer to remove the `requests` from resources.